tests/audio_test.py

```diff
@@ -25,3 +28,10 @@ def test_load_mp4_fails(monkeypatch):
     with pytest.raises(SystemExit) as sys_exit:
         audio_segment = load_mp3(original_name)
     assert 'The file must have .mp3 extension' in str(sys_exit)
+
+def test_get_jingles(monkeypatch):
+    song_name = 'song.mp3'
+    monkeypatch.setattr(AudioSegment, 'from_mp3', MockAudioSegment.from_mp3)
+    begin, end = get_jingles(song_name)
+    assert len(begin) == 20000
+    assert len(end) == 10000
```

Comment:
These assertions are not enough, since these only ensures the length of the resulting audio segments, not the content.